x-pack/legacy/plugins/ml/public/components/navigation_menu/main_tabs.tsx

```diff
@@ -66,7 +66,7 @@ interface TabData {
 }
 
 const TAB_DATA: Record<TabId, TabData> = {
-  // overview: { testSubject: 'mlTabOverview', pathId: 'overview' },
+  overview: { testSubject: 'mlTabOverview', pathId: 'overview' },
```

Comment:
The test subject naming has slightly changed with #45793. Would be good to have `mlMainTab overview` here to match the other tab's attributes.